superset/db_engine_specs/bigquery.py

```diff
@@ -396,10 +396,12 @@ def get_parameters_from_uri(
         raise ValidationError("Invalid service credentials")
 
     @classmethod
-    def mask_encrypted_extra(cls, encrypted_extra: str) -> str:
+    def mask_encrypted_extra(
+        cls, encrypted_extra: Union[str, None]
+    ) -> Union[str, None]:
         try:
-            config = json.loads(encrypted_extra)
-        except json.JSONDecodeError:
+            config = json.loads(encrypted_extra)  # type:ignore
```

Comment:
@betodealmeida the other option here would be to have a default value if encrypted_extra is None, but I opted for including it in the try/catch since it already existed. Of course, this requires the type ignores because the linter isn't aware of the try/except.

Reply:
Sounds good! Alternatively I think this would also work:

```python
def mask_encrypted_extra(cls, encrypted_extra: Optional[str]) -> Optional[str]:
    if encrypted_extra is None:
        return encrypted_extra

    try:
        ...
```

Reply:
Thanks for the suggestion. Yeah I think it looks better this way without the type ignores. I also left the TypeError in the except block just in case.